core/modules/simpletest/simpletest.install

```diff
@@ -13,33 +13,32 @@ define('SIMPLETEST_MINIMUM_PHP_MEMORY_LIMIT', '64M');
  * Implements hook_requirements().
  */
 function simpletest_requirements($phase) {
-  $requirements = array();
   $t = get_t();
+  $requirements = array();
 
+  // Only set a requirements message if cURL is disabled.
   $has_curl = function_exists('curl_init');
-  $has_hash = function_exists('hash_hmac');
-  $has_domdocument = method_exists('DOMDocument', 'loadHTML');
-  $open_basedir = ini_get('open_basedir');
-
-  $requirements['curl'] = array(
-    'title' => $t('cURL'),
-    'value' => $has_curl ? $t('Enabled') : $t('Not found'),
-  );
   if (!$has_curl) {
-    $requirements['curl']['severity'] = REQUIREMENT_ERROR;
-    $requirements['curl']['description'] = $t('The testing framework could not be installed because the PHP <a href="@curl_url">cURL</a> library is not available.', array('@curl_url' => 'http://php.net/manual/en/curl.setup.php'));
+    $requirements['simpletest_curl'] = array(
```

Comment:
Can we please add a `@todo` here to cycle back to this and merge it with the `$requirements['curl']` array provided by the System module, or hook-alter this once we have https://github.com/backdrop/backdrop-issues/issues/6241?